docs/source/markdown/podman-machine-init.1.md

```diff
@@ -76,15 +76,32 @@ Set the timezone for the machine and containers.  Valid values are `local` or
 a `timezone` such as `America/Chicago`.  A value of `local`, which is the default,
 means to use the timezone of the machine host.
 
-#### **--volume**, **-v**=*source:target*
+#### **--volume**, **-v**=*source:target[:options]*
 
 Mounts a volume from source to target.
 
 Create a mount. If /host-dir:/machine-dir is specified as the `*source:target*`,
 Podman mounts _host-dir_ in the host to _machine-dir_ in the Podman machine.
 
-The root filesystem is mounted read-only in the default operating system,
-so mounts must be created under the /mnt directory.
+Additional options may be specified as a comma-separated string. Recognized
+options are:
+* **ro**: mount volume read-only
+* **rw**: mount volume read/write (default)
+* **security_model=[model]**: specify 9p security model (see below)
+
+The 9p security model determines if and how the 9p filesystem translates some
```

Comment:
```suggestion
The 9p security model [determines](https://wiki.qemu.org/Documentation/9psetup#Starting_the_Guest_directly) if and how the 9p filesystem translates some
```

Reply:
Ok. I made this change along with rewrapping the surrounding text.